client/src/components/Login.vue

```diff
@@ -2,13 +2,12 @@
   <div class="login">
     <div class="container">
       <div class="row">
-        <div id="login-header">
-          <button id="login-btn" v-on:click="changeFormType($event)" v-bind:class="{'primary-btn': showingLoginForm, 'secondary-btn': showingRegisterForm}">Login</button>
-          <button id="register-btn" v-on:click="changeFormType($event)" v-bind:class="{'primary-btn': showingRegisterForm, 'secondary-btn': showingLoginForm}">Register</button>
-        </div>
-      </div>
-      <div class="row">
         <div id="login-form" class="column column-50 column-offset-25">
+          <div id="user-info-div" >
+              <span v-show="currentSlackRealname">Welcome, {{currentSlackRealname}}</span>
```

Comment:
I noticed that some people did not input their names in Slack as it's optional. Only their username is required. In this instance, `currentSlackRealname` would be `""` and this section wouldn't show.